src/app/map/[id]/components/PrivateMap.tsx

```diff
@@ -21,14 +23,21 @@ import PrivateMapNavbar from "./PrivateMapNavbar";
 import MapTable from "./table/MapTable";
 
 export default function PrivateMap() {
-  const { mapQuery, mapRef, showControls } = useContext(MapContext);
+  const { mapRef, showControls, mapId } = useContext(MapContext);
   const { areaStatsQuery, setLastLoadedSourceId } =
     useContext(ChoroplethContext);
 
   const { dataSourcesLoading } = useContext(DataSourcesContext);
   const { markerQueries } = useContext(MarkerAndTurfContext);
   const { selectedDataSourceId } = useContext(TableContext);
 
+  const trpc = useTRPC();
+  const { data: map, isPending } = useQuery(
+    trpc.map.byId.queryOptions(
+      { mapId: mapId || "" },
+      { enabled: Boolean(mapId) },
+    ),
+  );
```

Comment:
question: can we make this into a reusable hook, like `useMapQuery(mapId)` ? and put it in src/app/map/[id]/queries.ts ?